conversation_reconstruction/dataflow/analysis/calc_coordination.py

```diff
@@ -0,0 +1,68 @@
+import json
+import nltk
+import time
+from collections import defaultdict
+
+def get_words(content):
+    words = [liwc[w.lower()] for w in nltk.word_tokenize(content) if w.lower() in liwc]
+    ret = {m: 0 for m in markers}
+    for w in words:
+        for m in w:
+            ret[m] = 1
+    return ret
+ 
+
+markers = [10, 12, 18, 16, 3, 9, 17, 20]
```

Comment:
please comment magic numbers. maybe pass this into get_words() as a parameter to avoid the global reference.